lib/src/widget/input_history_text_field.dart

```diff
@@ -9,8 +10,7 @@ import 'package:input_history_text_field/src/widget/input_history_text_field_sta
 
 import '../model/input_history_item.dart';
 
-typedef HistoryListItemLayoutBuilder = Widget Function(
-    InputHistoryController controller, InputHistoryItem value, int index);
+typedef HistoryListItemLayoutBuilder = Widget Function(InputHistoryController controller, InputHistoryItem value, int index);
```

Comment:
Thanks for fixing it 🙆‍♂️ 

Sorry. I can't distinguish it from the intended bug fix 🙏 
Can you please remove the formatting fixes and just fix the bug fixes?

Other parts of the same, if you can get the same lint fixes to a different PR, that would be helpful.